contrib/scraper_joao_pessoa.py

```
@@ -0,0 +1,112 @@
+    poetry run scrapy runspider contrib/scraper_joao_pessoa.py -o output.jsonl
+
+Website: https://arquidiocesepb.org.br/
+Schedule page: https://arquidiocesepb.org.br/horario-de-missas-e-confissoes/
+"""
+
+import scrapy
+
+
+class JoaoPessoaSpider(scrapy.Spider):
+    name = "joao_pessoa"
+    allowed_domains = ["arquidiocesepb.org.br"]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)